src/database.c

```diff
@@ -115,16 +123,39 @@ log_database_init(ProfAccount* account)
         goto out;
     }
 
-    query = "CREATE TABLE IF NOT EXISTS `DbVersion` ( `dv_id` INTEGER PRIMARY KEY, `version` INTEGER UNIQUE)";
+    gboolean new_installation = (_get_db_version() == -1);
```

Comment:
It doesn't need to be a new installation. It can also be that it's just a a new user who gets a new database. So I would call it `first_run`, `first_start` or `db_not_existant` or.. I'm not sure :)

Reply:
Adressed